apiary.apib

```diff
@@ -5262,6 +5267,8 @@ Translates an ID from the deprecated API into a new UUID.
                     + creditNote
                     + subscription
                     + quotation
+                    + timeTracking
+                    + customField
```

Comment:
Q: Should be `customFieldDefinition` I think?

Reply:
It is.. 🙈